corehq/apps/app_manager/templates/app_manager/v1/form_view.html

```diff
@@ -107,7 +107,9 @@
     {# CloudCare #}
     {% initial_page_data 'app_id' app.id %}
     {% initial_page_data 'module_id' module.id %}
+    {% initial_page_data 'module_unique_id' module.unique_id %}
```

Comment:
Don't see why this is necessary to add, but it's v1 only anyway so not concerned about it.